Content.Client/GameTicking/Managers/TitleWindowManager.cs

```diff
@@ -15,48 +15,35 @@ public sealed class TitleWindowManager
 
     public void Initialize()
     {
-        _cfg.OnValueChanged(CVars.GameHostName, OnHostnameChange, true);
-        _cfg.OnValueChanged(CCVars.GameHostnameInTitlebar, OnHostnameTitleChange, true);
+        _cfg.OnValueChanged(CVars.GameHostName, _ => OnHostnameChange(), true);
+        _cfg.OnValueChanged(CCVars.GameHostnameInTitlebar, _ => OnHostnameChange(), true);
```

Comment:
This does not work for unsubbing.

The reason for this is that this makes a new lamba.